com.github.vikdevelop.photopea_app.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: photopea_app
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-download
+modules:
+  - shared-modules/dbus-glib/dbus-glib-0.110.json
+  - name: photopea_app
+    buildsystem: simple
+    build-commands:
+    - c++ photopea.cpp `pkg-config --cflags --libs gtk+-3.0 webkit2gtk-4.0` -o /app/bin/photopea_app
```

Comment:
`/app/bin` doesn't exist here.
Also it's a common practice to just output the binary in the current directory and then install it to its location afterward. I suggest doing the same here.

you don't have build scripts?